Assets/MRTK/Providers/Oculus/XRSDK/Controllers/OculusXRSDKTouchController.cs

```diff
@@ -33,6 +35,8 @@ public class OculusXRSDKTouchController : GenericXRSDKController
             : base(trackingState, controllerHandedness, inputSource, interactions, new OculusTouchControllerDefinition(controllerHandedness))
         { }
 
+        internal GameObject OculusControllerVisualization;
```

Comment:
suggestion on making this only privately settable:

```suggestion
        internal GameObject OculusControllerVisualization { get; private set; }
```